test/functionalTest/cases/2750_common_metrics/2750_incoming_errors.test

```diff
@@ -295,25 +302,44 @@ Date: REGEX(.*)
                     "serviceTime": REGEX(.*)
                 },
                 "SP/get/metrics": {
-                    "incomingTransactionResponseSize": 337,
+                    "incomingTransactionResponseSize": 457,
```

Comment:
Funny ... any change in Content-Length in these test cases is recursively spread to many other test cases ...
I would use the word nightmare, not fun ... :-D